.github/workflows/ihs-cisBuild.yml

```diff
@@ -0,0 +1,307 @@
+# This is a basic workflow to help you get started with Actions
+
+name: ihs-cis CICD
+
+# Controls when the action will run. 
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  # Allows you to run this workflow using GitHub APIs
+  # PERSONAL_ACCESS_TOKEN=<GITHUB_PERSONAL_ACCESS_TOKEN>
+  # REPO_NAME=WASdev/azure.websphere-traditional.image
+  # curl --verbose -XPOST -u "WASdev:${PERSONAL_ACCESS_TOKEN}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/${REPO_NAME}/actions/workflows/ihs-cisBuild.yml/dispatches --data '{"ref": "main"}'
+  repository_dispatch:
+    types: [integration-test-ihs-cis, integration-test-all]
+  # sample request
+  # PERSONAL_ACCESS_TOKEN=<GITHUB_PERSONAL_ACCESS_TOKEN>
+  # REPO_NAME=WASdev/azure.websphere-traditional.image
+  # curl --verbose -X POST https://api.github.com/repos/${REPO_NAME}/dispatches -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${PERSONAL_ACCESS_TOKEN}" --data '{"event_type": "integration-test-ihs-cis"}'
+  # curl --verbose -X POST https://api.github.com/repos/${REPO_NAME}/dispatches -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${PERSONAL_ACCESS_TOKEN}" --data '{"event_type": "integration-test-all"}'
+
+env:
+  repoName: azure.websphere-traditional.image
+  userName: ${{ secrets.USER_NAME }}
+  azureCredentials: ${{ secrets.AZURE_CREDENTIALS }}
+  entitledIbmUserId: ${{ secrets.ENTITLED_IBM_USER_ID }}
+  entitledIbmPassword: ${{ secrets.ENTITLED_IBM_USER_PWD }}
+  unEntitledIbmUserId: ${{ secrets.UNENTITLED_IBM_USER_ID }}
+  unEntitledIbmPassword: ${{ secrets.UNENTITLED_IBM_USER_PWD }}
```

Comment:
@majguo I appreciate you already including the necessary setup and teardown scripts.